src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-tutorials/opentelemetry-tutorial-java.mdx

```diff
@@ -0,0 +1,1132 @@
+    <img
+      title="Screenshot showing the total number of invocations of fibonacci function"
+      alt="Screenshot showing the total number of invocations of fibonacci function"
+      src={opentelemetryMetricsExplorerDimensions}
+    />
+
+Learn more about the metrics explorer view [here](docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-metrics-explorer-page).
+
+
+#### JVMs [#jvms]
+
+You can also view visualizations of your Java runtime metrics, including throughput, memory usage, and garbage collection time per minute, by heading to the JVM page:
+
+<img
+  title="Screenshot of the main JVMs page"
+  alt="Screenshot of the main JVMs page"
+  src={opentelemetryJvmsPageMain}
+/>
+
+You can view the metrics for a single instance, or select multiple instances to compare them:
+
+<img
+  title="Screenshot of how to compare JVM instances"
+  alt="Screenshot of how to compare JVM instances"
+  src={opentelemetryCompareJvmInstances}
+/>
+
+Here is what you will see after selecting your instances and clicking **Compare**, where each instance is color coded for easy identification:
+
+<img
+  title="Screenshot showing the color-coded JVM comparison"
+  alt="Screenshot showing the color-coded JVM comparison"
+  src={opentelemetryJvmColorCodedComparison}
+/>
+
+To learn more about the JVMs view, see [here](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-jvms-page/).
+
+### Logs [#logs]
+
+Here is where to access your logs:
+
+<img
+  title="Screenshot showing the main logs page"
+  alt="Screenshot showing the main logs page"
+  src={opentelemetryMainLogsPage}
+/>
+
+You will also see logs in your terminal:
+
+<img
+  title="Screenshot showing logs in a terminal"
+  alt="Screenshot showing logs in a terminal"
+  src={opentelemetryLogsinTerminal}
+/>
+
+Back in your logs view, select a log, and you will see a pane open up with the log message and additional attributes that were collected, such as the associated span and trace ids, as well as metadata injected by New Relic:
+
+<img
+  title="Screenshot showing log details with trace"
+  alt="Screenshot showing log details with trace"
+  src={opentelemetryLogDetailsWithTrace}
+/>
+
+You can navigate to the correlated distributed trace by clicking this little blue icon:
+
+<img
+  title="Screenshot showing how to find correlated trace"
+  alt="Screenshot showing how to find correlated trace"
+  src={opentelemetryTraceRelatedtoLog}
+/>
+
+This will open a pane that displays the correlated trace, where you can view more details about the trace. For more about this page, see [OpenTelemetry in the UI: Distributed tracing page](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-distributed-tracing-page/) and [Understand and use the distributed tracing UI](https://docs.newrelic.com/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui/):
```

Comment:
```suggestion
This will open a pane that displays the correlated trace, where you can view more details about the trace. For more about this page, see [OpenTelemetry in the UI: Distributed tracing page](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-distributed-tracing-page/) and [Understand and use the distributed tracing UI](/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui/):
```

Reply:
@zuluecho9 I did a search and replace for all these absolute paths! Thanks!